migrate.py

```diff
@@ -94,7 +94,8 @@ def main(options):
     """
     Main entry point for the script.
     """
-    bb_url = "https://api.bitbucket.org/1.0/repositories/{repo}/issues".format(
+    bb_url = "https://api.bitbucket.org/1.0/repositories/{account}/{repo}/issues".format(
+        account=options.bitbucket_username,
```

Comment:
This is incorrect. Reason being that I'll be adding support for private repos, and sometimes the bb username of a collaborator might be different from the repo's owner username.